backend/routes.js

```diff
@@ -279,7 +279,8 @@ export default class Servers {
                         status = 201;
```

Comment:
could we move this `existingUsers.push({...})` to the next section after the `if (contains(username, getSetting('ALLOWED_USERS') || that.isElectron || ...)`? We don't use `existingUsers` in any other case and without reading the next section, it almost looks like we're accepting _any_ login, not just those that are authorized.